mmeval/metrics/coco_detection.py

```diff
@@ -513,6 +504,8 @@ def compute_metric(self, results: list) -> Dict[str, Union[float, list]]:
         else:
             outfile_prefix = self.outfile_prefix
 
+        if self.classes is None:
+            self._get_classes()
```

Comment:
Reasonable modification!
However, should explain the reason here, in case others may try to 'optimize' this piece of code